### PR TITLE
(SERVER-910) Merge up 1.x

### PR DESCRIPTION
**Pull request by dankreek**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ group :test do
   gem 'rspec'
   gem 'beaker', *location_for(ENV['BEAKER_VERSION'] || '~> 2.14.0')
   if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
-    gem 'sqa-utils', '~> 0.11'
+    gem 'sqa-utils', '0.12.1'
```
Comment (camlow325) [inline Gemfile]: @waynr, @nwolfe - This version of `sqa-utils` seem okay for testing from Puppet Server's stable branch - 2.x series / compatible with Puppet 4.2.Z series?  I'm totally ignorant about `sqa-utils`.

**Comment by camlow325** [comment]
:+1: with just the one question about the `sqa-utils` version.

File: Gemfile (hunk 1 above)
Comment (nwolfe) [inline Gemfile]: Seems okay to me but I'll defer to @waynr

**Comment by camlow325** [comment]
:shipit:

File: Gemfile (hunk 1 above)
Comment (waynr) [inline Gemfile]: I generally prefer to use `~> <MAJOR>.<MINOR>` to ensure bug fixes and minor platform support updates are included but in the next sprint or two I am hoping to convert all teams away from `sqa-utils`/`genconfig2` to the new `beaker-hostgenerator` gem.

**Pull request merged**